tool/bazelinstall/rbe.sh

```diff
@@ -43,8 +43,8 @@ if [[ -n "$BAZEL_BUILDBUDDY_CERT" && -n "$BAZEL_BUILDBUDDY_KEY" ]]; then
     echo "The RBE credential has been installed!"
     echo "Configuring Python..."
     # setting the exact version of Python 3 and Python 2, respectively
-    pyenv install 3.6.10
-    pyenv global 3.6.10 2.7.12
+    pyenv install -s 3.6.10
```

Comment:
Can we instead do:

```
pyenv install -s 3.6.10
pyenv install -s 2.7.12
pyenv global 3.6.10 2.7.12
```

Reply:
@lriuui0x0 we could, but `python` (Python 2) is installed system-wide and we don't really care which one it is - we're going to move off depending on Python 2 soon anyway

Reply:
OK cool @vmax